src/terms/term_external.jl

```diff
@@ -78,6 +78,7 @@ function term_external(generators_or_composition...; compensating_background=tru
         if energy !== nothing
             dVol = model.unit_cell_volume / prod(basis.fft_size)
             energy[] = real(sum(ρ.real .* Vext)) * dVol
+            # @assert energy[] ≈ real(dot(ρ.fourier, coeffs)) # sanity check
```

Comment:
Remove that?

Reply:
I think it's nice to have these kind of invariants explicit somewhere, makes debugging and figuring out the code much simpler. Maybe with an expensive assert?

Reply:
Yes. I think that's a good idea.